app/src/main/res/values/strings.xml

```diff
@@ -387,7 +389,7 @@
     <string name="toast_field_value_to_long">Field value too long : %s</string>
     <string name="toast_field_value_to_short">Field value too short : %s</string>
     <string name="toast_field_must_be_not_empty">Field must be not empty : %s</string>
-    <string name="title_activity_main_meny">MainMeny</string>
+    <string name="title_activity_main_meny">Main Menu</string>
```

Comment:
remove this and everything what related with lables or titels should be in text_view_field folder, never don't use string.xml